README.md

```diff
@@ -10,100 +13,91 @@ This repository contains the reference implementation of the Enterprise Data Str
 
-## Providers
+The target destination is configured using the `--url` option. The provider that is selected is based on the scheme/protocol part of the URL. The following providers are supported:
 
-The following are the supported providers:
+- **mysql** - used to stream data into a MySQL database
+- **postgres** - used to stream data into a PostgreSQL database
+- **snowflake** - used to stream data into a Snowflake database
+- **s3** - used to stream data into a S3 compatible cloud storage (AWS, Google Cloud, Minio, etc)
+- **kafka** - used to stream data into a Kafka topic
 
-- [PostgreSQL DB](https://www.postgresql.org/)
-- [SQL Server DB](https://www.microsoft.com/en-us/sql-server)
-- [Snowflake DB](https://www.snowflake.com/)
-- File - use `file://<PATH>` to stream Json lines via STDIN to an executable provided by PATH
-- Nats Provider. Use the --nats-provider setting and it'll allow you to connect a nats consumer to the local NATS server and process messages via consumer
-  See [Nats Consumer Example](./examples/python/nats-provider.py)
+> Not all provider's currently support importing data
 
-## Development
+You can get a list of providers with example URL patterns by running the following:
 
-These are the instructions to run EDS for local development. This will spin up a postgres and azure-edge database using docker compose.
+```
+./eds-server server help
+```
 
-```bash
-./hack/localstack
+You can get specific help for a provider with the following:
 
-export SQL_PASS=Asdf1234! && go run . server --creds ../location/to/credential-file "sqlserver://sa:$SQL_PASS@localhost:1433?database=shopmonkey"
+```
+./eds-server server help [provider]
+```
 
-export PGPASS=postgres && go run . server --creds ../location/to/credential-file "postgresql://postgres:$PGPASS@localhost:5432/shopmonkey?sslmode=disable"
+Such as:
 
+```
+./eds-server server help s3
 ```
 
-To run EDS with Snowflake, your connection string should be in the format of:
-
-`snowflake://<username>:<password>@<organization_name>-<account>/<database_name>/<schema>?warehouse=<warehouse_name>&client_session_keep_alive=true`
+## Importing Data
 
-The `client_session_keep_alive=true` portion is optional, but you may run into authentication issues after 4 hours if there is no activity from EDS to Snowflake. See [Snowflake Session Policies](https://docs.snowflake.com/en/user-guide/session-policies) for more details.
+The import command will take a snapshot of the data in the Shopmonkey Database and import that data to your destination. This is useful to bootstrap or backfill a new destination with existing data. You can then run the server to stream additional incremental changes as they happen.
 
-A full connection string would look like: `snowflake://jsmith:mypassword@zflycky-cu81015/mydb/PUBLIC?warehouse=COMPUTE_WH&client_session_keep_alive=true`
+Example:
 
-## Importer
+```
+./eds-server import --api-key 123 --url "postgresql://admin:root@localhost:5432/test?sslmode=disable"
```

Comment:
small change, but v3 uses `postgres://` now instead of `postgresql://` as the connection url prefix,

Reply:
yeah, we were trying to clean it up because pgx uses something different than pq driver and i didn't want to specifically tie to a driver in case we want to change. i will make sure it uses postgres is the driver prefix but that its always make it postgresql when it makes the internal connection string.

Reply:
ok, i pushed some cleanup around this and test case.